diff -up libvosk-0.3.50/CMakeLists.txt.omv~ libvosk-0.3.50/CMakeLists.txt
--- libvosk-0.3.50/CMakeLists.txt.omv~	2024-05-21 13:52:41.535382400 +0200
+++ libvosk-0.3.50/CMakeLists.txt	2024-05-21 13:54:44.225477635 +0200
@@ -10,10 +10,12 @@ add_library(vosk
   src/recognizer.cc
   src/spk_model.cc
   src/vosk_api.cc
+  src/postprocessor.cc
 )
 
 find_package(kaldi REQUIRED)
-target_link_libraries(vosk PUBLIC kaldi-base kaldi-online2 kaldi-rnnlm fstngram)
+# FIXME is there a proper way to link to ngram-fst and if so, what is it?
+target_link_libraries(vosk PUBLIC kaldi-base kaldi-online2 kaldi-rnnlm -Wl,-rpath,@LIBDIR@/fst @LIBDIR@/fst/ngram-fst.so)
 
 include(GNUInstallDirs)
 install(TARGETS vosk DESTINATION ${CMAKE_INSTALL_LIBDIR})
